edx_django_utils/monitoring/internal/utils.py

```diff
@@ -143,3 +157,22 @@ def noop_decorator(func):
         return newrelic.agent.background_task(*args, **kwargs)
     else:
         return noop_decorator
+
+
+@contextmanager
+def monitor_django_management_command(name):
+    """
+    A context manager for monitoring Django management commands.
+
+    This function creates a monitoring span using `function_trace`, allowing
+    the execution of the command to be tracked explicitly in monitoring tools.
+    It also sets the transaction name to the given `name` using
+    `set_monitoring_transaction_name`.
```

Comment:
I don't want the second point to get lost. What about?: 
```suggestion
    - This function creates a monitoring span using `function_trace`, allowing
      the execution of the command to be tracked explicitly in monitoring tools.
    - It also sets the transaction name to the given `name` using
      `set_monitoring_transaction_name`.
```